app/controllers/api/v1/classrooms_controller.rb

```diff
@@ -4,7 +4,7 @@ class Api::V1::ClassroomsController < Api::V1::BaseController
 
   def index
     @classrooms = policy_scope(Classroom)
-    render json: @classrooms.to_json(methods: :class_time)
+    render json: @classrooms.map(&:parse_for_dashboard).to_json
```

Comment:
you removed the other method that was being passed, try to implement your method in the same way

Reply:
Got it. Added the class time to the parse_for_dashboard method.